Banking/src/main/java/com/example/Banking/Exception/IdNotFound.java

```diff
@@ -0,0 +1,21 @@
+package com.example.Banking.Exception;
+import lombok.Data;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Data
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+@Component
+
+public class IdNotFound extends Exception {
```

Comment:
Is it used anywhere?

Reply:
this is also part of exception. by mistake committed exception files. removed them now.